common/blindedpath.c

```diff
@@ -147,38 +147,15 @@ static bool blind_node(const struct privkey *blinding,
 	return true;
 }
 
-bool blinded_node_alias(const struct privkey *blinding,
```

Comment:
Looks like this commit could be merged with the prior one.

Reply:
Yes, I forgot to fold it in.  Fixing.